.golangci.yml

```diff
@@ -73,6 +73,7 @@ linters-settings:
       - name: error-strings
       - name: error-return
       - name: receiver-naming
+      - name: increment-decrement
```

Comment:
@jwcesign have you analyzed how many rules we don't need to fix?
Can we enable them with a single PR?

Reply:
Around 8.

> Can we enable them with a single PR?

Sure.

Reply:
Go for it. It makes no sense for contributors to just add a rule without the fix.
/lgtm
/approve